src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py

```diff
@@ -276,3 +310,91 @@
     type: command
     short-summary: Place the CLI in a waiting state until a condition of the VM is met.
 """
+
+helps['disk'] = """
+    type: group
+    short-summary: Commands to manage 'Managed Disks'
+"""
+
+helps['snapshot'] = """
+    type: group
+    short-summary: Commands to manage snapshots
+"""
+
+helps['image'] = """
+    type: group
+    short-summary: Commands to manage custom virtual machine images based on managed disks/snapshots
+"""
+
+helps['disk create'] = """
```

Comment:
Is this effectively a first-class replacement for manually creating/formatting/uploading a VHD to a blob container? If so, that is so amazing!